server/src/services/recaptcha/recaptcha.service.ts

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios'
+import { createLogger } from '../../bootstrap/logging'
+import { errAsync, okAsync, ResultAsync } from 'neverthrow'
+import { recaptchaConfig } from '../../bootstrap/config/recaptcha'
+
+import { GOOGLE_RECAPTCHA_URL } from './recaptcha.constants'
+import {
+  CaptchaConnectionError,
+  MissingCaptchaError,
+  VerifyCaptchaError,
+} from './recaptcha.errors'
+
+const logger = createLogger(module)
+export const verifyCaptchaResponse = (
```

Comment:
For consistency, we should have a RecaptchaService which accepts an axios client in its ctor. This would allow you to avoid the module mocking in your tests. Since you would also be injecting the RecaptchaService into the EnquiryController, you can also provide unit-testing for the controller and even the route (especially given that you now have supertest).

For convenience, the ctor can also accept just the Google ReCAPTCHA URL; you then construct an axios client from that

Reply:
Going with the first suggestion of passing an axios into RecaptchaService